jena-arq/pom.xml

```diff
@@ -123,9 +123,18 @@
       <groupId>com.jayway.awaitility</groupId>
       <artifactId>awaitility</artifactId>
       <scope>test</scope>
+    </dependency><dependency>
```

Comment:
This is totally confusing. Not only does it appear that you are pulling in the same dependency twice, this very dependency is already pulled in at line 76 above!